morpheus/app/views.py

```diff
@@ -308,7 +308,17 @@ class DeleteSubaccountView(ServiceView):
         r = await mandrill.post('subaccounts/delete.json', allowed_statuses=(200, 500), id=m.company_code, timeout_=12)
         data = await r.json()
         if r.status == 200:
-            return PreResponse(text='subaccount deleted\n', status=200)
+            async with self.app['pg'].acquire() as conn:
+                del_messages_count = await conn.execute(
+                    'delete from messages m using message_groups mg where m.group_id=mg.id and mg.company=$1',
+                    m.company_code,
+                )
+                del_groups_count = await conn.execute('delete from message_groups where company=$1', m.company_code)
+            data = {
```

Comment:
This is a very dumb way of doing things, why do we create a dict, then convert it to JSON in the output.

You want something like:

```py
deleted_message_count = int(del_messages_resp.replace('DELETE ', ''))
del_groups_count = int(del_groups_resp.replace('DELETE ', ''))
msg = f'deleted_messages={deleted_message_count} deleted_message_groups={del_groups_count}'
logger.info('deleting company=%s %s', m.company,msg)
return PreResponse(text=msg + '\n', status=200)
```